modules/features/podcasts/src/main/java/au/com/shiftyjelly/pocketcasts/podcasts/view/components/ratings/GiveRatingScreen.kt

```diff
@@ -0,0 +1,198 @@
+import androidx.compose.ui.graphics.vector.ImageVector
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.tooling.preview.PreviewParameter
+import androidx.compose.ui.unit.dp
+import au.com.shiftyjelly.pocketcasts.compose.AppThemeWithBackground
+import au.com.shiftyjelly.pocketcasts.compose.bars.NavigationButton
+import au.com.shiftyjelly.pocketcasts.compose.bars.NavigationIconButton
+import au.com.shiftyjelly.pocketcasts.compose.buttons.RowButton
+import au.com.shiftyjelly.pocketcasts.compose.components.PodcastCover
+import au.com.shiftyjelly.pocketcasts.compose.components.TextH30
+import au.com.shiftyjelly.pocketcasts.compose.preview.ThemePreviewParameterProvider
+import au.com.shiftyjelly.pocketcasts.compose.theme
+import au.com.shiftyjelly.pocketcasts.podcasts.viewmodel.GiveRatingViewModel
+import au.com.shiftyjelly.pocketcasts.podcasts.viewmodel.GiveRatingViewModel.State.Loaded.Stars
+import au.com.shiftyjelly.pocketcasts.ui.theme.Theme
+import au.com.shiftyjelly.pocketcasts.localization.R as LR
+
+@Composable
+fun GiveRatingScreen(
+    state: GiveRatingViewModel.State.Loaded,
+    setStars: (Stars) -> Unit,
+    submitRating: () -> Unit,
+    onDismiss: () -> Unit,
+) {
+
+    val context = LocalContext.current
+    Content(
+        state = state,
+        setStars = { setStars(it) },
+        submitRating = {
+            Toast.makeText(context, "TODO: Submit rating", Toast.LENGTH_LONG).show()
+            submitRating()
+        },
+        onDismiss = onDismiss,
+    )
+}
+
+@Composable
+private fun Content(
+    state: GiveRatingViewModel.State.Loaded,
+    setStars: (Stars) -> Unit,
+    submitRating: () -> Unit,
+    onDismiss: () -> Unit,
+) {
+    Column(
+        horizontalAlignment = Alignment.CenterHorizontally,
+        modifier = Modifier.fillMaxSize()
+    ) {
+        NavigationIconButton(
+            iconColor = MaterialTheme.theme.colors.primaryText01,
+            navigationButton = NavigationButton.Close,
+            onNavigationClick = onDismiss,
+            modifier = Modifier.align(Alignment.Start)
+        )
+
+        Spacer(Modifier.weight(1f))
+
+        Column(
+            horizontalAlignment = Alignment.CenterHorizontally,
+        ) {
+
+            PodcastCover(
+                uuid = state.podcastUuid,
+                coverWidth = 164.dp,
+            )
+
+            Spacer(Modifier.height(40.dp))
+
+            TextH30(text = stringResource(LR.string.podcast_rate, state.podcastTitle))
+
+            Spacer(Modifier.height(32.dp))
+
+            StarsRow(
+                stars = state.stars,
+                setStars = setStars,
+            )
+        }
+
+        Spacer(Modifier.weight(1f))
+
+        RowButton(
+            text = stringResource(LR.string.submit),
+            onClick = submitRating
+        )
+    }
+}
+
+@Composable
+private fun StarsRow(
```

Comment:
This `StarsRow` composable is just a placeholder. I'll be replacing it and the functionality around selecting stars in a follow-up PR.